CONTRIBUTING.rst

```diff
@@ -0,0 +1,76 @@
+Releasing
+=========
+
+Summary
+-------
+
+- Create a release branch in the main repository, not in a fork.
+- Prepare a commit for a release candidate, get an approving review, and tag the commit which will upload artifacts to PyPI.
+- Notify people and give time for feedback.
```

Comment:
Where to notify? I have no idea where to notify. Maybe IRC #twisted-dev  ?

How much time? -- Maybe 1 week.

Reply:
```suggestion
- Notify the Twisted maillist allow a week for feedback before continuing.
```